algorithms/src/sorting/impl/Kokkos_SortByKeyImpl.hpp

```diff
@@ -189,6 +189,21 @@ void applyPermutation(const ExecutionSpace& space,
       KOKKOS_LAMBDA(int i) { view(i) = view_copy(permutation(i)); });
 }
 
+// Use functors as nvcc errors out on trying to get the address of a function
```

Comment:
I think this comment should mention this is an issue specifically with lambdas inside a variadic function under nvcc. As the comment reads now, making it a lambda should fix things, yet that is the original problem.

(Other than that, I'm fine with this.)

Reply:
I'm not sure how to properly describe it, though. Lambdas seem to work with nvcc when the execution space is also on the device. It's only when the stuff is on the host that it becomes a problem. So, I don't quite know the full extent of the problem. How would you formulate the comment?

Reply:
@nliber I updated the comment.